pkg/services/quota/quota.go

```diff
@@ -151,6 +151,12 @@ func (qs *QuotaService) getQuotaScopes(target string) ([]models.QuotaScope, erro
 			models.QuotaScope{Name: "global", Target: target, DefaultLimit: qs.Cfg.Quota.Global.Session},
 		)
 		return scopes, nil
+	case "alert_rule": // target need to much the respective database name
```

Comment:
Is this comment necessary to keep? I guess it should say "match" in case it should stay